package.json

```diff
@@ -4,17 +4,16 @@
   "description": "Yet another Base64 transcoder in pure-JS",
   "main": "base64.js",
   "module": "base64.mjs",
-  "browser": "base64-es5.js",
```

Comment:
@dallonf Would you like to publish a patch version? The incorrect `browser` field may break some user cases of bundlers.

Reply:
Did you mean to tag @dankogai?